lib/charms/opensearch/v0/opensearch_plugin_manager.py

```diff
@@ -138,11 +138,19 @@ def check_plugin_manager_ready(self) -> bool:
         """Checks if the plugin manager is ready to run."""
         return (
             self._charm.opensearch.is_node_up()
```

Comment:
Let's add this as a first check:
```
self._charm.peers_data.get(Scope.APP, "security_index_initialised", False)
```

This should save us a few subsequent checks when the first cluster spins up, or when the large relation data hasn't completely been populated.